### ISSUE TITLE
Q16 does not show up in questionnaire

### ISSUE THREAD

**Issue by touwwouter** [DataStewardshipWizard/dsw-common#72]
[ds-km core question 16](https://github.com/DataStewardshipWizard/ds-km/blob/8d42657bc44fd8cba8ce30f508878aff763c2435/datamodel/core/chapter1.json#L218) ('Root Questionnaire (Common ELIXIR Knowledge Model, 1.0.0) `"Do you need to harmonize different sources of existing data?"`) does not show up when the questionnaire is filled using the DataStewardshipWizard/dsw-client hosted at https://app.dsw.fairdata.solutions. If I understand the datamodel correctly, it should appear when the answer to question 2 `"Will you be using any pre-existing data (including other people's data)?"` is `"Yes"`:

```
...
	{
	    "questionid": 2, "type": "option",
	    "precondition": { "questionid": 1, "answerid": 1 },
	    "title": "Will you be using any pre-existing data (including other people's data)?",
	    "text": "Will you be referring to any earlier measured data, reference data, or data that should be mined from existing literature? Your own data as well as data from others?",
	    "answers": 
	    [
		{   "id": 0, "label": "No",
		    "advice": "Did you research all the data that exists? You may not be aware of all existing data that could be available. Although using and/or integrating existing data sets may pose a challenge, it will normally be cheaper than collecting everything yourself. Even if you decide not to use an existing data set, it is better to do this as a conscious decision." },
		{   "id": 1, "label": "Yes" }
	    ],
	    "references": [{"type": "dmpbook", "chapter": "1.2"}, {"type": "resourcepage", "shortuid": "ezi"}]
	},
...
	{
	    "questionid": 16, "type": "option",
	    "precondition": { "questionid": 2 },
	    "title": "Do you need to harmonize different sources of existing data?",
	    "text": "If you are combining data from different sources, harmonization may be required. You may need to re-analyse some original data.",
	    "answers":
	    [
		{   "id": 0, "label": "No" },
		{   "id": 1, "label": "Yes" }
	    ],
	    "references": [{"type": "dmpbook", "chapter": "1.10"}, {"type": "resourcepage", "shortuid": "wht"}]
	},
...
```

Is this due to dsw-client frontend logic or the ds-km schema?

**Comment by MarekSuchanek** [comment]
Thanks for reporting, I will look at it. Maybe there is some bug when I was rewriting it half-automatically from old IDs to UUIDs.

We really need to merge that asap...

**Comment by MarekSuchanek** [comment]
I see... This is problem of this older KM support in new Wizard. It does allows just followups bound to answer and now whole questions and to list items. I can not fix it now just in km2events so it will add question-followup to all options, because there can't be 2 (or more) paths for 1 followup...

We need to think about how to implement it in Wizard so it will work also for single-field questions... @vknaisl @janslifka 

Honestly, I missed that there is such case in KM already...

**Comment by vknaisl** [comment]
No longer needed (solved by #87)
